proposals/navigator-identity-proposal.md

```diff
@@ -1,3 +1,9 @@
+# ⚠️ This proposals has been superseded ⚠️
+
+- Please refer to the [Digital identities spec](https://wicg.github.io/digital-identities/) instead. It represents the community's consensus.
+
+## Alternate design
+
 This is an alternative design that optimizes for decoupling identity verification and authentication further (so that developers get a clear signal about their differences) while still accomplishing the goal of being able to request multiple identity schemes (e.g. mdocs, vcs and federation) at the same time in the same dialog.
```

Comment:
```suggestion
This is an alternative design that further optimizes the decoupling of identity verification and authentication (so that developers get a clear signal about their differences), while still accomplishing the goal of being able to request multiple identity schemes (e.g., mdocs, vcs, and federation) at the same time in a single dialog.
```